### ISSUE TITLE
Support for parallel compilation

### ISSUE THREAD

**Issue by nipunn1313** [alexcrichton/gcc-rs#90]
Would be nice to compile files in parallel (equivalent to `make -j4` for example).

**Comment by alexcrichton** [comment]
Fixed by https://github.com/alexcrichton/gcc-rs/pull/92

**Comment by nipunn1313** [comment]
Can you up the version number to include this?

**Comment by alexcrichton** [comment]
Sure! Pushed 0.3.33

**Comment by nipunn1313** [comment]
Thanks!

**Comment by nipunn1313** [comment]
I tried applying the features flag to this project (https://github.com/nipunn1313/rust-rocksdb), but didn't seem to get any speedup. I confirmed that the rayon code path is being taken with a parallelism of 4, but I don't seem to see 4 tasks getting spun out concurrently. I confirmed that the majority of time is spent in the "compile_objects" function rather than the "assemble" function as well.

Still trying to get to the bottom of it, but I thought I'd see if you had any quick insight on this one.
